### DISCUSSION TITLE
Change display for pass fail status

### DISCUSSION THREAD

**Discussion by jdwilkin4** [jdwilkin4/contrast-ratio-repo#120] [Ideas]
For this version 1 release, I think it would be better to just show pass fail text instead of the svg icons.
For future releases we could change it to use the icons. 
Something similar to this, where it will conditionally render the results for AA and AAA standards based on contrast ratio. 
**Note: I didn't mess with the graphics section since that is already being worked on in another issue**

<img width="726" alt="Screen Shot 2022-09-04 at 2 49 09 PM" src="https://user-images.githubusercontent.com/67210629/188334818-0baa8e11-f4c5-4156-b35e-0cbfe4bd6595.png">

Thoughts about making this change and creating a ticket for it?

**Comment by sparkO7** [comment]
I agree, so should we change the HTML and replace all svg elements to something like spans. 
What about the "icons" class should we change it to something else?

**Reply by jdwilkin4** [reply]
I was playing around with it earlier and it doesn't look like we need the icons class. 
We could get rid of those icons divs and styles all together. 
We could always add it back in later for future versions when the svgs come back in. 

On page load the design would look something like this
<img width="760" alt="Screen Shot 2022-09-04 at 3 30 19 PM" src="https://user-images.githubusercontent.com/67210629/188335861-db6a2bf5-c561-4805-97bd-dfd0f961f0a5.png">


Then when the user inputs values it would result in this
<img width="791" alt="Screen Shot 2022-09-04 at 3 32 16 PM" src="https://user-images.githubusercontent.com/67210629/188335916-1a5dd602-88ee-4c54-bcb7-bc4baf4bf484.png">

**Reply by sparkO7** [reply]
I see. So, the AA+/AAA+ labels too will only show when the result shows. And the pass/fail are just texts with borders, right?

**Reply by jdwilkin4** [reply]
Yeah, that was the plan

**Reply by jdwilkin4** [reply]
I went ahead and created an ticket for it
https://github.com/DBenMoshe/contrast-ratio-repo/issues/126
